packages/google_maps_flutter/google_maps_flutter_web/pubspec.yaml

```
@@ -22,10 +22,11 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  google_maps: ^6.1.0
+  google_maps: ^7.0.0
   google_maps_flutter_platform_interface: ^2.4.0
   sanitize_html: ^2.0.0
   stream_transform: ^2.0.0
+  web: ">=0.3.0 <0.5.0"
```

```suggestion
  web: ^0.5.0
```

We should now move to the latest and greater here!

---

Well, `google_maps` will have to upgrade as well then. cc @a14n 

```
Resolving dependencies... (1.2s)
Because google_maps 7.0.0 depends on web ^0.4.0 and no versions of google_maps match >7.0.0 <8.0.0, google_maps ^7.0.0 requires web ^0.4.0.
So, because google_maps_flutter_web_integration_tests depends on both web ^0.5.0 and google_maps ^7.0.0, version solving failed.
```

I can send a patch out for that, though.

---

thank you!

---

This is unblocked in google_maps, so I'll update this PR tomorrow with the new constraint.

---

Done!